common/supplemental/supplementalData.xml

```diff
@@ -4787,91 +4787,91 @@ XXX Code for transations where no currency is involved
-              <era type="0" start="622-7-15" code="islamic-tbla" aliases="ah"/>
+              <era type="0" start="622-7-15" code="ah"/> <!-- Anno Hegirae -->
           </eras>
         </calendar>
         <calendar type="islamic-umalqura">
           <calendarSystem type="lunar" />
           <eras>
-              <era type="0" start="622-7-15" code="islamic-umalqura" aliases="ah"/>
+              <era type="0" start="622-7-15" code="ah"/> <!-- Anno Hegirae -->
           </eras>
         </calendar>
         <calendar type="chinese">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-2636-01-01" code="chinese"/> <!-- 'sequential' year -->
+              <era type="0" start="-2636-01-01"/> <!-- 'sequential' year -->
           </eras>
         </calendar>
         <calendar type="hebrew">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-3760-10-7" code="hebrew" aliases="am"/>
+              <era type="0" start="-3760-10-7" code="am"/> <!-- Anno Mundi -->
           </eras>
         </calendar>
         <calendar type="buddhist">
           <calendarSystem type="solar" />
           <eras>
-              <era type="0" start="-542-01-01" code="buddhist" aliases="be"/>
+              <era type="0" start="-542-01-01" code="be"/> <!-- Buddhist Era -->
           </eras>
         </calendar>
         <calendar type="coptic">
           <calendarSystem type="other"/>
           <eras>
-              <era type="0" end="284-08-28" code="coptic-inverse"/>
-              <era type="1" start="284-08-29" code="coptic"/>
+              <era type="0" end="284-08-28" code="bd"/> <!-- Before Diocletian. NOTE: No evidence this era is used -->
+              <era type="1" start="284-08-29" code="am" primary="true"/> <!-- Anno Martyrum -->
           </eras>
         </calendar>
         <calendar type="persian">
           <calendarSystem type="solar"/>
           <eras>
-              <era type="0" start="622-01-01" code="persian" aliases="ap"/>
+              <era type="0" start="622-01-01" code="ap" aliases="sh hs"/> <!-- Anno Persico -->
           </eras>
         </calendar>
         <calendar type="dangi">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-2332-01-01" code="dangi"/> <!-- 'sequential' year -->
+              <era type="0" start="-2332-01-01"/> <!-- 'sequential' year -->
           </eras>
         </calendar>
         <calendar type="ethiopic">
           <calendarSystem type="other"/>
           <eras>
-              <era type="0" end="8-08-28" code="ethioaa" aliases="ethiopic-amete-alem mundi"/>
-              <era type="1" start="8-08-29" code="ethiopic" aliases="incar"/>
+              <era type="0" end="8-08-28" code="aa" aliases="mundi"/> <!-- Amete Alem -->
+              <era type="1" start="8-08-29" code="am" aliases="incar" primary="true"/> <!-- Amete Mihret -->
```

Comment:
consider removing the aliases, they're our invention afaict

Reply:
Our invention or not, if an ID has been in any release, it needs to be deprecated but maintained as an alias.

Reply:
In theory yes, but if neither ICU, nor ICU4X, nor Temporal, nor any implementation we know of has used these, can we be a bit flexible?

Reply:
I'd like to discuss this question in the CLDR Design WG meeting. As a general principle I agree with @macchiati, but I don't have evidence that these codes are used _anywhere_, so I might prefer making an exception in this one case.

Reply:
> I don't have evidence that these codes are used _anywhere_, so I might prefer making an exception in this one case.

Well, it is really the opposite question. Unless we have proof they are they are *not* used anywhere I think we should keep the old codes as aliases. We do not necessarily know who all of the clients of CLDR data are and how they use it.

Reply:
We discussed this in the 2025-03-31 CLDR Design WG meeting and the conclusion from that meeting (based on the people who happened to be in attendance that day) was that given how news these codes are, and given that ICU does not use them, and Temporal doesn't use them yet, and that they were added _for Temporal_, they are new enough that we can just drop the codes currently in the XML.

But I do not want to block this PR on that discussion so if adding the existing codes as aliases is going to move this forward, I'll do that